<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>正規紙庫存系統</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="paperInputTitle">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>到港未入庫</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="systemStatusComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>到港未入庫</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>正規紙庫存</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>正規紙排單</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>排單剩餘</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="paperInputPushButton">
      <property name="text">
       <string>輸入</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>systemStatusComboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>paperInputTitle</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
